docs/performance/Basics.md

```diff
@@ -87,7 +87,7 @@ As the drop in price is accounted for by the dividend represented by `ExpectedIn
 
 ## Partial Withdrawals
 
-Another scenario that could cause this is that of a partial withdrawal.
+Another scenario that could cause this erroneous fluctuation in performance is that of a partial withdrawal.  Without considering Expected Income it looks like this.
```

Comment:
Reads a bit clunky to me, should we just add empty Expected Income to the table?

Reply:
Agreed it's clunky, but I think it might be better to just remove the example table altogether, and get straight to the 'good' example showing the calculation with Expected Income in its table.